Formula/safe.rb

```diff
@@ -0,0 +1,33 @@
+class Safe < Formula
+  desc "Vault CLI"
+  homepage "https://github.com/starkandwayne/safe"
+  url "https://github.com/starkandwayne/safe/archive/v0.9.0.tar.gz"
+  sha256 "98b6e4aca9ca294c99f9a7e6b5d72175c9af044ec91a59430857c8619a7ad883"
+
+  depends_on "go" => :build
+  depends_on "vault"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["VERSION"] = version
+
+    (buildpath/"src/github.com/starkandwayne/safe").install buildpath.children
+
+    cd "src/github.com/starkandwayne/safe" do
+      system "make", "build"
+      bin.install "safe"
```

Comment:
You'll need a `prefix.install_metafiles` line underneath this now this is the final stage of the installation process. It's a known piece of fun with `go` stuff in Homebrew.

Reply:
fixed!  and as a bonus i learned about `brew ls --verbose` 😁 

btw i really appreciate the blow-by-blow review of this formula; i'm learning a lot.